libbeat/publisher/pipeline/module.go

```diff
@@ -127,6 +129,10 @@ func loadOutput(
 	}
 
 	if !outcfg.IsSet() {
+		if !outRequired {
+			return outputs.Group{}, nil
```

Comment:
We could still log something, in case someone configures beats using CM, but still without an output.

Reply:
+1 for some debug/info log here

Reply:
Thanks for your input folks, pushed 34bba0d7f, I struggled a bit with the messaging here, I'm open for feedback on it

Reply:
I'd add just an info line mentioning that current loaded configuration has no outputs and then no data will be published. So if after configuring modules someone find that they still don't publish anything, they can find a clue to their mistake in the logs.

Reply:
This is the current message: `No outputs are configured, starting in paused mode.`, wdyt?

Reply:
Looks like you added the Info log to the wrong line.

Reply:
Umm, you added it to the dry-run flow, was this intended?

Regarding the message, I was more concerned to the case of the user changing the config using CM but not adding any output (or removing this by mistake if possible), so not only when the beat starts, also I wouldn't talk about a "paused mode" (do we have this concept?).
What about something that announces the effect to the user, like `No outputs are configured, no data will be collected`?

Reply:
I want to avoid something that will make the user think this is bad, as it will be there if you are using central management. Best message to me would be something like this: 

`Output settings are centrally managed, go to Kibana to configure one`

Now, I understand we don't want to be that specific from the `outRequired` flag, so I'm seeking for a message that explains that as soon as you see CM logs after it

Reply:
`Output configuration pending`?